config/default.js

```diff
@@ -349,6 +350,8 @@ module.exports = {
 
   enableFeatureInlineAddonReview: true,
 
+  enableFeatureNewHomeHero: false,
```

Comment:
Maybe you can call this feature `HomeHeroGuides`. Otherwise we might end up with a `NewNewHomeHero`, etc. next time.